lnwallet/channel.go

```diff
@@ -1155,11 +1164,51 @@ func (lc *LightningChannel) SignNextCommitment() ([]byte, uint32, error) {
 	return sig, lc.theirLogCounter, nil
 }
 
-// ReceiveNewCommitment processs a signature for a new commitment state sent by
+// Validate is used to validate that on current state the commitment
```

Comment:
Nice, I like the addition of this function! It can later be further generalized to catch things like enforcement of the channel "reserve" as defined in the spec, the limits on the total value of HTLC's in flight, etc. 

Also I don't think this method needs to be publicly exported.

One minor nit is that I think the function needs a more descriptive name, how about something like `validateCommitmentThresholds` or `validateCommitmentSanity`?